src/main.js

```diff
@@ -20,9 +25,22 @@ This will make sure Firebase initializes before loading the app when a user refr
 - https://savvyapps.com/blog/definitive-guide-building-web-app-vuejs-firebase
 */
 
+let config = appConfig;
+// Sample setup for ImageSwipe and WordSwipe for development for testing only. Will be removed when testing is done
+if (process.env.VUE_APP_PROJECT_ID === "imageswipe-test") {
+  config = imageswipeTestConfig;
+}
+if (process.env.VUE_APP_PROJECT_ID === "wordswipe-test") {
```

Comment:
you could probably put this logic in the `config.js` and just have that file export the correct one